src/types.ts

```diff
@@ -249,6 +251,9 @@ export interface PSESponsorshipsFormData extends CaptchaForm {
   eventBudgetBreakdown: string; // SF API: Proposed_Timeline__c
   eventRequestedAmount: string; // SF API: Sponsorship_Monetary_Request__c
   additionalInfo: string; // SF API: Additional_Information__c
+  // Optional fields for Quadratic Funding Initiative option
```

Comment:
Looking at the preview and code, it looks like these two fields are for both options now, right?

Reply:
Right